crates/matrix-sdk-ui/Cargo.toml

```diff
@@ -8,6 +8,7 @@ rust-version = { workspace = true }
 default = ["e2e-encryption", "native-tls"]
 
 e2e-encryption = ["matrix-sdk/e2e-encryption"]
+experimental-oidc = ["matrix-sdk/experimental-oidc"]
```

Comment:
As said somewhere else, we don't use Cargo features for the UI crate.

Reply:
Well, not when it's just for things being experimental, at least (since the crate itself is rather unstable).